src/mol-model-formats/structure/property/bonds/struct_conn.ts

```diff
@@ -117,13 +120,16 @@ export namespace StructConn {
             // turns out "mismat" records might not have atom name value
             if (!atomName) return undefined;
 
+            // prefer auth_seq_id, but if it is 0, then fall back to label_seq_id
+            const resId = ps.auth_seq_id.value(row) ? ps.auth_seq_id.value(row) : ps.label_seq_id.value(row);
```

Comment:
I still don't know how I feel about this. `model.atomicHierarchy.index.findResidue` explicitly expects `auth_seq_id`, so this will only work if the file you are loading has either undefined `atom_site.auth_seq_id` (in which case the `label_` version will be used automatically for this) or if `auth_seq_id` <=> `label_seq_id` everywhere.

---------

Also, I can see the edge case where someone complains that the struct_conn doesn't work for auth seq id 0 which I am sure someone somewhere used.

Reply:
Use `.valueKind` on the column. https://github.com/molstar/molstar/blob/master/src/mol-data/db/column.ts#L108

Reply:
Ok, I'll switch to check on valueKind. Btw, in the OpenMM files, atom site auth and label seq ids are the same.